static/js/components/UserMenu_test.js

```diff
@@ -9,13 +9,29 @@ import { makeUser } from "../factories/user"
 
 describe("UserMenu component", () => {
   const user = makeUser()
+  it("have MixedLinks for login/registration for mobile view", () => {
```

Comment:
This test case is not testing what the description says. We would not expect login/registration links because the user is authenticated. Also, `useScreenOverlay={false}` so this is not testing the mobile view.

As this test is written, an accurate description would be something like this:
```suggestion
  it("has the correct number of menu links", () => {
```